docs/md/expressions.md

```diff
@@ -0,0 +1,429 @@
+
+### Numbers
+
+* General Math: `min, max, avg, sum, abs, ceil, floor, round, round, exp, log,
+  log10, log1p, log2, logn, pow, root, sqrt`
+* Trigonometry: `sin, cos, tan, acos, asin, atan, atan2, cosh, cot, csc, sec,
+  sinh, tanh, deg2rad, rad2deg, deg2grad, grad2deg`
+* `iclamp(min, value, max)` - If `value` is between `min` and `max`, then the
+  function produces the number to which `value` is closest, using `min` if the
+  distance is equal. If `value` is not in range, then `value` is used as-is.
+* `inrange(min, value, max)` - Produces `true` or `false` depending on whether
+  `value` is between `min` and `max` (inclusive).
+* `percent_of(value, total)` - Produce the ratio between the value and total.
+* `random()` - Produce a random number between 0.00 and 1.00.
+* `frac(x)` - Fractional portion (after the decimal) of `x`
+
+### Time
+
+* `date(year, month, day)` - Generate a date.
+* `datetime(timestamp)` - Generate a datetime, where `timestamp` is [Unix
+  time](https://en.wikipedia.org/wiki/Unix_time) in milliseconds.
+* `today()` - Present date.
+* `now()` - Present datetime.
+* `hour_of_day(x)` - Return a datetime's hour of the day as a string.
+* `day_of_week(x)` - Return a datetime's day of week as a string.
+* `month_of_year(x)`- Return a datetime's month of the year as a string.
+
+### Type Conversions
+
+* `integer(x)` - Convert `x` to an integer number.
+* `float(x)` - Convert `x` to a decimal number.
+* `boolean(x)` - Convert `x` to true/false.
+* `string(x)` - Convert `x` to text.
+
+### Misc
+
+* `is_null(x)` - Is `true` if `x` has no value.
+* `is_not_null(x)` - Is `true` if `x` has a value.
+* `order("input column", 'value', ...)` - Generates a sort order for a string
+  column based on the input order of the parameters, such as: `order("State",
+  'Texas', 'New York')`. This would produce the following output:
+
+  | State       | My Column    |
+  | ----------- | ------------ |
+  | Texas       | 0.00         |
+  | New York    | 1.00         |
+  | Connecticut | 2.00         |
+  | Maine       | 2.00         |
+  | Oregon      | 2.00         |
+
+<!-- TODO: Could not make this work. -->
+<!-- * `not(x)` -->
+
+### Bucket
+
+`bucket(x, y)` - Bucket value `x` by `y`, where `y` is dependent on `x`'s value
+type.
+
+#### Bucketing by Number
+
+`bucket("Sales", 100)` - Produces number bucketed by increments of 100 (always
+rounded down).
+
+<div>
+<perspective-viewer
+  columns='["Sales", "My Column Name"]'
+  expressions='["// My Column Name\n\nbucket(\"Sales\", 100)"]'
+></perspective-viewer>
+</div>
+
+#### Bucketing by Date
+
+`bucket("Order Date", 'D')` - Produces date from datetime.
+`bucket("Order Date", 'W')` - Produces date from the first day of that week.
+`bucket("Order Date", 'M')` - Produces date from the first day of that month.
+`bucket("Order Date", 'Y')` - Produces date from the first day of that year.
+
+## Conditional Structures
+
+Acting based on conditions is similar to most programming languages.
```

Comment:
Core ExprTK - may want to look at how the ExprTK docs describe this feature also as I think it's a bit clearer, especially if you're coming form an Excel background rather than a Python (or other programming) background.